apps/website/src/pages/ambassadors/[ambassadorName].tsx

```diff
@@ -311,8 +311,8 @@ const AmbassadorPage: NextPage<AmbassadorPageProps> = ({
       </div>
 
       {ambassador.clips.length > 0 && (
-        <Section dark>
-          <Heading level={2} className="text-center text-4xl">
+        <Section dark className="bg-alveus-green-800">
+          <Heading id="highlights" level={2} className="text-center text-4xl">
```

Comment:
Is it worth also setting the `link` prop here, so folks can click the heading to get the anchor?

Reply:
sure